processor/src/test/java/org/mapstruct/ap/test/inheritfromconfig/multiple/CarMapperTest.java

```diff
@@ -0,0 +1,105 @@
+/**
+ *  Copyright 2012-2017 Gunnar Morling (http://www.gunnarmorling.de/)
+ *  and/or other contributors as indicated by the @authors tag. See the
+ *  copyright.txt file in the distribution for a full listing of all
+ *  contributors.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package org.mapstruct.ap.test.inheritfromconfig.multiple;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.Date;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mapstruct.ap.testutil.IssueKey;
+import org.mapstruct.ap.testutil.WithClasses;
+import org.mapstruct.ap.testutil.runner.AnnotationProcessorTestRunner;
+
+@RunWith(AnnotationProcessorTestRunner.class)
+@WithClasses({
+    BaseDto.class,
+    BaseEntity.class,
+    CarDto.class,
+    CarEntity.class,
+    CarMapper.class,
+    EntityToDtoMappingConfig.class,
+    Car2Dto.class,
+    Car2Entity.class,
+    CarMapper2.class
+})
+@IssueKey("1367")
+public class CarMapperTest {
+
+    @Test
+    public void testMapEntityToDto() {
+        CarDto dto = CarMapper.MAPPER.mapTo( newCar() );
+        assertThat( dto.getDbId() ).isEqualTo( 9L );
+        assertThat( dto.getMaker() ).isEqualTo( "Nissan" );
+        assertThat( dto.getSeatCount() ).isEqualTo( 5 );
+    }
+
+    @Test
+    public void testMapDtoToEntity() {
+        CarEntity car = CarMapper.MAPPER.mapFrom( newCarDto() );
+        assertThat( car.getId() ).isEqualTo( 9L );
+        assertThat( car.getManufacturer() ).isEqualTo( "Nissan" );
+        assertThat( car.getNumberOfSeats() ).isEqualTo( 5 );
+
+        // following assertion fails :  createdBy property is not set.
```

Comment:
Do we still need this line and the one below?

Reply:
I'll remove that one.. You're right..